index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&family=Source+Sans+Pro:ital,wght@0,400;0,700;1,400;1,700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <script src="https://kit.fontawesome.com/b2291adafa.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <title>Mobile version</title>
+</head>
+<body>
+ <section class="main-container">
+    <Section class="toolbar">
+        <Nav>
+            <h3>My Logo</h3>
+            <span class="material-icons">menu</span>
+        </Nav>
+    </Section>
+    <section class="headline">
+            <h1>I'm Esther <br>Glad to see you!</h1>
+            <p>Lorem ipsum dolor sit amet consectetur adipisicing elit. Sapiente harum nostrum maxime saepe ad.,  Incidunt aut eaque voluptatibus doloribus atque debitis voluptatum itaque veniam praesentium earum. Molestiae perferendis beatae est?</p>
+            <h4>LET'S CONNECT</h4>
+    </section>
+    <section class="icons">
+            <a href=""><img src="../Imgs/Vector.svg" alt=""></a>  
+            <a href=""><img src="../Imgs/linkedin.svg" alt=""></a>
+            <a href=""><img src="../Imgs/M-Vector.svg" alt=""></a>
+            <a href=""><img src="../Imgs/github.svg" alt=""></a>
+            <a href=""><img src="../Imgs/Victory.svg" alt=""></a>
+            
+    </section>
+</section>
```

Comment:
- [ ]  Your design should look much similar to the design from provided Figma design.  Adjust your font size, family, and weight referring to the fonts provided from Figma design. 

Your design | Figma design
--- | ---
![image](https://user-images.githubusercontent.com/43877689/170741422-1f119191-497f-4b18-a78f-51287b809dc9.png) | ![image](https://user-images.githubusercontent.com/43877689/170741712-982bca15-8b2f-40c3-ac0a-672ee3d6f90f.png)